v2/tools/generator/internal/astmodel/std_properties.go

```diff
@@ -18,10 +18,34 @@ const (
 	OperatorSpecConfigMapsProperty   = "ConfigMaps"
 	ConditionsProperty               = "Conditions"
 	OptionalConfigMapReferenceSuffix = "FromConfig"
+	UserAssignedIdentitiesProperty   = "UserAssignedIdentities"
+	UserAssignedIdentitiesTypeName   = "UserAssignedIdentityDetails"
 )
 
 // IsKubernetesResourceProperty returns true if the supplied property name is one of the properties required by the
 // KubernetesResource interface.
 func IsKubernetesResourceProperty(name PropertyName) bool {
 	return name == AzureNameProperty || name == OwnerProperty
 }
+
+func IsUserAssignedIdentityProperty(prop *PropertyDefinition) (TypeName, bool) {
+	if !prop.HasName(UserAssignedIdentitiesProperty) {
+		return TypeName{}, false
```

Comment:
```suggestion
		return astmodel.EmptyTypeName, false
```

And other places too.

Reply:
Done